### PR TITLE
Feature: Transfer list on finance page & Transfer page DAO-751 

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

- Transfer lists on the finance and transfer page are connected to the graph.
- some helper functions were added from date-fns for having the human-readable date format
- refactored transfer list item based on graph and designs

Task: [dao-751](https://aragonassociation.atlassian.net/jira/software/c/projects/DAO/boards/2?modal=detail&selectedIssue=DAO-751)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/hooks/useCategorizedTransfers.tsx [hunk 1]
```diff
@@ -19,7 +21,15 @@ export type CategorizedTransfer = {
...
-export default function useCategorizedTransfers(): HookData<CategorizedTransfer> {
+export default function useCategorizedTransfers(): HookData<CategorizedTransfer> & {
+  totalTransfers: string;
+} {
+  const {data: daoTransfers} = useDaoTransfers(
+    '0x51c3ddb42529bfc24d4c13192e2e31421de459bc'
+  );
+  const {data: transfers, total} = usePollTransfersPrices(daoTransfers);
+
+  console.log('transfers', total);
```
Comment (Fabricevladimir) [inline packages/web-app/src/hooks/useCategorizedTransfers.tsx]: Can you remove the logs and unnecessary comments?

File: packages/web-app/src/hooks/useTokenMetadata.tsx [hunk 2]
```diff
@@ -23,6 +23,8 @@ export const useTokenMetadata = (balances: TokenBalance[]) => {
         )
       );
 
+      console.log('seeMeta', metadata, balances);
```
Comment (Fabricevladimir) [inline packages/web-app/src/hooks/useTokenMetadata.tsx]: Don't forget to remove.

File: packages/web-app/src/pages/finance.tsx [hunk 3]
```diff
@@ -11,48 +11,20 @@ import TokenList from 'components/tokenList';
...
-
 const Finance: React.FC = () => {
   const {t} = useTranslation();
   const {open} = useGlobalModalContext();
-  const {tokens, totalAssetChange, totalAssetValue} = useDaoVault(
+  const {tokens, totalAssetChange, totalAssetValue, transfers} = useDaoVault(
     '0x51c3ddb42529bfc24d4c13192e2e31421de459bc'
   );
 
+  console.log('transfers', transfers);
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/finance.tsx]: And here as well.

File: packages/web-app/src/hooks/usePollTransfersPrices.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+import {ASSET_PLATFORMS} from 'utils/constants';
+import {DaoTransfer, Transfer} from 'utils/types';
+import {formatUnits} from 'utils/library';
+import {formatDate} from 'utils/date';
+
+export const usePollTransfersPrices = (transfers: DaoTransfer[]) => {
+  const client = useApolloClient();
+  const {chainId} = useWallet();
+  const [data, setData] = useState<Transfer[]>([]);
+  const [total, setTotal] = useState<string>('');
```
Comment (Fabricevladimir) [inline packages/web-app/src/hooks/usePollTransfersPrices.tsx:15]: Make sure that this total is not different from the Vault total.

**Review by Fabricevladimir — COMMENTED** [review]
Thanks! Good work, hopefully this will mitigate refactors when the SDK is ready to go

**Review by RakeshUP — APPROVED** [review]

**Review by Fabricevladimir — APPROVED** [review]

**Pull request merged**